### ISSUE TITLE
Cannot scroll list of books with NVDA

### ISSUE THREAD

**Issue by sukiletxe** [edrlab/thorium-reader#1276]
I have 28 books added to my library (or at least, 28 folders in the publications-dev folder). Only some of them are shown in Thorium. When I press on "scroll list to the left" or "scroll list to the right" the button changes, but not the library list itself. I have tried refreshing the page with NVDA+F5 but nothing happens.

v1.6.0-alpha.1.400550802

**Comment by sukiletxe** [comment]
The other presentation also fails to show all books, only shows 19 (11 in the resume reading list and 8 in the recently added list).

**Comment by danielweck** [comment]
> The other presentation also fails to show all books, only shows 19 (11 in the resume reading list and 8 in the recently added list).

This is a known UX "bug" (i.e. confusing user interface). See https://github.com/edrlab/thorium-reader/issues/1273

**Comment by danielweck** [comment]
Thank you for reporting the NVDA bug. Just out of interest: are you able to test with other Windows screen readers as well? (i.e. JAWS and Narrator) Do you have access to a MacOS computer to test VoiceOver?

**Comment by sukiletxe** [comment]
Just tested with arrator, the result is the same. To clarify, I don't know if the list updates visually or not, I just know that more buttons appear (if I press scroll right scroll left appears). I cannot test with other screen readers

**Comment by sukiletxe** [comment]
fixed with the fix of #1273. Thanks!
